test/helper/test_timeline.js

```diff
@@ -303,15 +303,19 @@ const testTimelineFiles = (timelineFiles) => {
         it('should have proper sealed sync', () => {
           for (const sync of timeline.syncStarts) {
             const regex = sync.regex.source;
+            // FIXME: This test will currently accept log lines with or without a second colon,
+            // as "0839:" or "0839::".
+            // Once we have completely converted things for 6.0,
+            // we should come back here and make the doubled colon non-optional.
             if (regex.includes('is no longer sealed')) {
               assert(
-                regex.includes('00:0839:.*is no longer sealed'),
-                `${timelineFile}:${sync.lineNumber} 'is no longer sealed' sync must be exactly '00:0839:.*is no longer sealed'`,
+                /00:0839::?\.\*is no longer sealed/.exec(regex),
```

Comment:
```suggestion
                /00:0839::?.*is no longer sealed/.exec(regex),
```

Reply:
Oh, that's fair, I guess there might be edge cases where we do want to permit specific unseal lines, as opposed to the One True Normalized one.

Reply:
Oh oh oh, you're right.  I think it should be like it was before I made a mistaken edit.  Sorry, I shouldn't send code review comments when i'm tired.

Reply:
It is a *good* thing for all of us that tomorrow is Saturday and not Monday!